src/views/modals/ModalTransactionCosignature/ModalTransactionCosignatureTs.ts

```diff
@@ -317,7 +319,8 @@ export class ModalTransactionCosignatureTs extends Vue {
                 if (c) {
                     return (
                         c.plain() === this.currentAccount.address ||
-                        (this.multisigAccountGraph && MultisigService.isAddressInMultisigTree(this.multisigAccountGraph, c.plain()))
+                        (this.multisigAccountGraph && MultisigService.isAddressInMultisigTree(this.multisigAccountGraph, c.plain())) ||
+                        (this.rootMultisigAccountGraph && MultisigService.isAddressInMultisigTree(this.rootMultisigAccountGraph, c.plain()))
                     );
```

Comment:
consider test for new branch

Reply:
Thanks. I found another issue, working on fix now. I will reopen PR later.